.travis.yml

```diff
@@ -3,35 +3,25 @@
 language: node_js
 matrix:
   include:
+    # Test on latest version of node
+    - node_js: 'node'
 
-    # Run lint only in Node.js 0.12
-    - node_js: '0.12'
+    # Run lint only in the latest LTS
+    - node_js: '6'
```

Comment:
This doesn't seem consistent.  Here you're saying `6` for latest Node, and above you're saying `node`.